packages/cli/changelog.md

```diff
@@ -4,14 +4,83 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next
+
+### Added
+- Contributing guidelines ([#553](https://github.com/zeppelinos/zos/pull/553))
+- Example using truffle migrations ([#416](https://github.com/zeppelinos/zos/pull/416))
+- Integration tests using `truffle-hdwallet-provider` ([#473](https://github.com/zeppelinos/zos/pull/473))
+- Tests for `verify` command ([#468](https://github.com/zeppelinos/zos/pull/468))
+- Migration process for projects using previous versions of ZeppelinOS v2 ([#621](https://github.com/zeppelinos/zos/pull/621))
+- Support compiling with globally installed Truffle ([#597](https://github.com/zeppelinos/zos/pull/597))
+- Support receiving booleans via the CLI ([#623](https://github.com/zeppelinos/zos/pull/623))
+- Parse scientific notation numbers from the CLI ([#567](https://github.com/zeppelinos/zos/pull/567))
+- Linter pre-commit githook ([#620](https://github.com/zeppelinos/zos/pull/620))
+
+### Changed
+- Update Web3 to version `1.0.0-beta.37` ([#572](https://github.com/zeppelinos/zos/pull/572))
```

Comment:
```suggestion
- **Update Web3 to version `1.0.0-beta.37`** ([#572](https://github.com/zeppelinos/zos/pull/572))
```

Reply:
Isn't this actually part of the lib changelog?

Reply:
I put it in both changelogs since cli is also depending on web3 1.x